inc/class-comment-popularity.php

```diff
@@ -97,6 +100,11 @@ private function __construct() {
 	}
 
 	public function styles() {
+		wp_enqueue_style(
```

Comment:
Not keen on loading this entire font for 3 icons only. I think it's best to use some custom CSS to override the existing look & feel if needed.